src/DacpacTool/DacpacTool.csproj

```diff
@@ -13,8 +13,8 @@
 
   <ItemGroup>
     <!-- These packages contain DacFX analysis rules, which must be located with the executable in order to be discovered-->
-    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0-preview4" IncludeAssets="runtime" />
-    <PackageReference Include="ErikEJ.DacFX.TSQLSmellSCA" Version="1.0.0-preview1" IncludeAssets="runtime" />
+    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0-preview7" IncludeAssets="runtime" />
```

Comment:
Do we now still want to ship these in the box?

Reply:
I think so, they (in particular the SqlServer.Rules) add quite some value - based on experience from production project